WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/InPersonPaymentsMenuViewModel.swift

```diff
@@ -1,18 +1,22 @@
 import Foundation
 import Yosemite
+import protocol Storage.StorageManagerType
 
 class InPersonPaymentsMenuViewModel: ObservableObject {
```

Comment:
Should this class be final?

Reply:
Yeah I can't see why not, will change in a follow-up PR 😊